projects/core/src/occ/adapters/product/occ-product.adapter.ts

```diff
@@ -26,6 +27,33 @@ export class OccProductAdapter implements ProductAdapter {
     protected requestsOptimizer: OccRequestsOptimizerService
   ) {}
 
+  loadRealTimeStock(
+    productCode: string,
+    unit: string
+  ): Observable<{ quantity: string; availability: string }> {
```

Comment:
can we create an data model for productavailabilites and use it here?

Reply:
Done